src/main/kotlin/com/hibob/academy/dao/OwnerDao.kt

```diff
@@ -21,11 +21,11 @@ class OwnerDao @Inject constructor(private val sql: DSLContext) {
         )
     }
 
-    fun createOwner(ownerName: String, companyId: Long, employeeId: String): Long {
+    fun createOwner(ownerInfo: Owner): Long {
```

Comment:
why did you change it?
If you have Owner meaning it was already created (have an id), you can use parameters or create a different data class for creation